lib/room.rb

```diff
@@ -1,12 +1,27 @@
-class Rom
+class Room
+
+  attr_reader :category
 
   def initialize(category, length, width)
     @category = category
-    @length = 10
+    @length = length
     @width = width
+    @paint = nil
```

Comment:
we'd probably set this to false (i.e., when a house is very first created, it is unpainted).  This is logical, and additionally, it makes it a little easier since you don't need the conditional in your `is_painted?` method.